_plugins/version.rb

```diff
@@ -0,0 +1,52 @@
+# Aggregate the En lesson versions together
+Jekyll::Hooks.register :site, :post_read do |site|
+  versions = site.pages.reduce(Hash.new { |h, k| h[k] = [] }, &method(:page_versions))
+  site.config["versions"] = versions
+end
+
+# Compare a page's version to the master version and populate a warning if necessary
+Jekyll::Hooks.register :pages, :pre_render do |page, payload|
+  data           = page.data
+  category       = data["category"]
+  default_lang   = page.site.config["default_lang"]
+  lang           = data["lang"]
+  order          = data["order"].to_i
+  master_version = page.site.config["versions"][category][order]
+  page_version   = data["version"] || "0.0.0"
```

Comment:
Once @gemantzu's page updates are on this branch, the default versions here and on line 46 can probably go away